README.md

```diff
@@ -250,5 +251,151 @@ mstorage.meta("<media_id>", new CompletionHandler<MediaMeta>() {
 });
 ```
 
+### Get specific media metadata
+
+* Exif
+
+```java
+mstorage.meta("<media_id>", "exif", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> exif) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Google Photo Sphere XMP
+
+```java
+mstorage.meta("<media_id>", "gpano", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> exif) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* All user metadata
+
+```java
+mstorage.meta("<media_id>", "user", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> userMeta) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+* Specific user metadata
+
+```java
+mstorage.meta("<media_id>", "user.<key>", new CompletionHandler<MediaMeta>() {
+    @Override
+    public void onCompleted(Map<String, String> userMeta) {
+        // Do something.
+    }
+
+    @Override
+    public void onThrowable(Throwable t) {
+        // Something wrong happened.
+    }
+});
+```
+
+### Attach media metadata
+You can define your original metadata as a 'user metadata'.
+
```

Comment:
Did you mean to add the following sentence?
"Existing metadata value for the same key will be overwritten. Up to 10 user metadata can be attached to a media data. "